MediaRecorder.html

```diff
@@ -442,6 +443,38 @@
               <em>Return type:</em> <code>boolean</code>
             </div>
           </dd>
+
+          <dt><dfn><code>getSupportedTypes</code></dfn>, static</dt>
+          <dd>
+            When <code>getSupportedTypes()</code> method is invoked, the UA MUST
+            queue a task, using the DOM manipulation task source, that runs the
+            following steps:
+            <ol>
+            <li>Let <var>p</var> be a promise.</li>
+            <li>Run the following steps in parallel:
+             <ol>
+             <li>Let <var>resultList</var> be an empty list.
+             <li>For each <a href="https://tools.ietf.org/html/rfc2046">MIME
```

Comment:
RFC2045 says a couple of things:
> [...] the remainder of the header field is simply a set of parameters, specified in an attribute=value notation.  The ordering of parameters is not significant.
 
Also, 

>[...] MIME implementations must ignore any parameters whose names they do not recognize.

Note that `video/webm` and `audio/webm` are not officially assigned by IANA ([here](http://www.iana.org/assignments/media-types/media-types.xhtml#video) is the list), so the only definition comes, informally, from the [webm web page](http://www.webmproject.org/docs/container/).